libs/openFrameworks/graphics/ofPath.cpp

```diff
@@ -59,6 +59,22 @@ ofPath::ofPath(){
 	clear();
 }
 
+//----------------------------------------------------------
+ofPath::ofPath(const ofPolyline &polyline) : ofPath(){
+	addPolyline(polyline);
+}
+
+//----------------------------------------------------------
+void ofPath::addPolyline(const ofPolyline &polyline){
+	if (mode == COMMANDS) {
+		for (ofVec3f vertex : polyline) {
```

Comment:
Maybe we can use `auto vertex` instead of `ofVec3f vertex`, and later just `lineTo(vertex)`. This should make this method more independent of the vertex type.